src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalRent = days * 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return totalRent - minDiscount;
+  } else if (days >= 7) {
+    return totalRent - maxDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 3 && days < 7) {
    return totalRent - minDiscount;
  } 
  
  if (days >= 7) {
    return totalRent - maxDiscount;
  }
```

just FYI - better to avoid `else if`. it improves readability